Systems/Auction/Auction.cs

```diff
@@ -325,8 +325,6 @@ private void Event_PlayerDisconnect(IServerPlayer byPlayer)
         {
             if (createAuctionSlotByPlayer.TryGetValue(byPlayer.PlayerUID, out var inv))
             {
-                inv.DropAll(byPlayer.Entity.Pos.XYZ);

```

Comment:
this is not necessary, as it is called when the inventory is closed

https://github.com/anegostudios/vssurvivalmod/blob/afc94069d30e0285185f5bf2dd798fe1b99fc2eb/Systems/Auction/Auction.cs#L346-L348